workers/docker-worker/src/lib/task.js

```diff
@@ -129,12 +132,33 @@ function runAsPrivileged(task, allowPrivilegedTasks) {
   return true;
 }
 
-async function buildDeviceBindings(devices, expandedScopes) {
-  let allowed = await hasPrefixedScopes('docker-worker:capability:device:', devices, expandedScopes);
```

Comment:
I'm really happy to see a use of `hasPrefixedScopes` go away.  Removing it from `buildVolumeBindings` will be a decent good-first-bug.